ui/lib/apps/DashboardSettings/translations/zh.yaml

```diff
@@ -0,0 +1,5 @@
+dashboard_settings:
```

Comment:
We no longer have this file, you can remove it

Reply:
I see, you removed dashboard setting to the user profile page in previous commit. I will remove it.

Reply:
Push a new commit https://github.com/pingcap-incubator/tidb-dashboard/pull/677/commits/436817df5977987d29bbfb89da4c03801a5db880 to remove the dashboard setting page.